react/Viewer/components/ToolbarFilePath.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+import { useClient } from 'cozy-client'
+
+import Link from '../../Link'
+import AppLinker from '../../AppLinker'
+import FilePath from '../../FilePath'
+import useBreakpoints from '../../hooks/useBreakpoints'
+
+import { useFileWithPath } from '../hooks/useFileWithPath'
+import { removeFilename, makeParentWebLink } from '../helpers'
+
+const ToolbarFilePath = ({ file }) => {
+  const client = useClient()
+  const { isDesktop } = useBreakpoints()
+
+  const fileWithPath = useFileWithPath(file)
+
+  if (fileWithPath) {
+    const app = 'drive'
+    const nativePath = `/folder/${fileWithPath.dir_id}`
+    const path = removeFilename(fileWithPath.path)
+    const link = makeParentWebLink({ client, nativePath, app })
```

Comment:
Is this function correctly named? It generates parent web link just because you pass parent nativePath no?

Reply:
I've renamed the function to `makeWebLink` and made the parameters more generic. Does this seem more clear to you?